features/cmake/find_package/exported_targets_multiconfig/README.md

```diff
@@ -0,0 +1,61 @@
+Using unobtrusive multi-config CMake with exported targets & find_package
+=========================================================================
+
+In this example
+---------------
+
+We are using CMake to declare, export and install the targets of a couple of libraries, and using Conan to manage
+the packages and using these libraries in a project. Everything is based on the ``find_package`` feature and working with 
+multiconfig packages (Release/Debug) to avoid installing the dependencies every time we change from Debug to Release in
+the Visual Studio IDE.
+
+ - We have a "hello" library consumed by a "bye" library and a project that uses the "bye" function.
```

Comment:
I think it is not clear at all what is the "project" recipe, and who is including who. (project->hello). 

I also think that the CMakeLists.txt of "project" should probably be added here in the readme, to see the transparent integration.